File: src/main/java/it/posteitaliane/omp/UI/view/HomeView.java
package it.posteitaliane.omp.UI.view;

import com.vaadin.navigator.ViewChangeListener;
import com.vaadin.ui.VerticalLayout;

public class HomeView extends BaseView<VerticalLayout> {
    public HomeView() {
        super(new VerticalLayout());
        setSizeFull();
    }

    @Override
    protected void onEnter(ViewChangeListener.ViewChangeEvent event) {
    }


}

File: src/main/scala/it/posteitaliane/omp/scalaUI/view/HomeView.scala
package it.posteitaliane.omp.scalaUI.view

import com.vaadin.ui.VerticalLayout
import com.vaadin.navigator.ViewChangeListener.ViewChangeEvent

class HomeView extends VerticalLayout with BaseView {
  setSizeFull()

  def onEnter(event: ViewChangeEvent) {}
}
